minos/networks/handlers/decorators.py

```diff
@@ -0,0 +1,90 @@
+# Copyright (C) 2020 Clariteia SL
+#
+# This file is part of minos framework.
+#
+# Minos framework can not be copied and/or distributed without the express
+# permission of Clariteia SL.
+
+
+class BrokerCommandEnroute:
+    """Broker Command Enroute class"""
+
+    _topics: list[str]
+
+    def __init__(self, *args, **kwargs):
+        self._topics = kwargs["topics"]
```

Comment:
we suppose that topics kwargs must be a list and we have to check and support that topics arg would be a string or a list

Reply:
Also, It would be better to directly set `topics: list[str]` in the function definition instead of extracting it from `kwargs`

Reply:
Done!